frontend/src/App.tsx

```diff
@@ -60,50 +60,54 @@ function MobileAppToolbar() {
   }
 
   return (
-    <Toolbar>
-      <IconButton onClick={() => setOpen(true)}>
-        <MenuIcon />
-      </IconButton>
-      <Drawer variant="persistent" anchor="left" open={open}>
-        <Button onClick={() => setOpen(false)}>
-          <ChevronLeftIcon />
-        </Button>
-        <List>
-          {Object.keys(PAGE_URL_TO_NAMES).map((pageUrl, index) => (
-            <ListItemLink href={pageUrl} key={index}>
-              <ListItemText primary={PAGE_URL_TO_NAMES[pageUrl]} />
-            </ListItemLink>
-          ))}
-        </List>
-      </Drawer>
-    </Toolbar>
+      <Toolbar>
+        <IconButton onClick={() => setOpen(true)}>
+          <MenuIcon />
+        </IconButton>
+        <Drawer variant="persistent" anchor="left" open={open}>
+          <Button aria-label="Hide site navigation button"
+                  onClick={() => setOpen(false)}>
+            <ChevronLeftIcon />
+          </Button>
+          <List>
+            {Object.keys(PAGE_URL_TO_NAMES).map((pageUrl, index) => (
+                <ListItemLink href={pageUrl} key={index}
+                              aria-label={PAGE_URL_TO_NAMES[pageUrl]}>
+                  <ListItemText primary={PAGE_URL_TO_NAMES[pageUrl]} />
+                </ListItemLink>
+            ))}
+          </List>
+        </Drawer>
+      </Toolbar>
   );
 }
 
 function AppToolbar() {
   return (
-    <Toolbar className={styles.AppToolbar}>
-      <LinkWithStickyParams to="/">
-        <img src="img/AppbarLogo.png"
-             className={styles.AppbarLogoImg}
-             alt="Health Equity Tracker Logo -- decorative 'H'"/>
-      </LinkWithStickyParams>
-      <Typography variant="h6" className={styles.HomeLogo}>
+      <Toolbar className={styles.AppToolbar}>
         <LinkWithStickyParams to="/">
-          Health Equity Tracker
+          <img src="img/AppbarLogo.png"
+               className={styles.AppbarLogoImg}
+               alt="Health Equity Tracker Logo -- decorative 'H'"/>
```

Comment:
I would consider the logo decorative (no announcement needed) and use alt="". Since the title is announced just prior, the screen reader would announce, "Health Equity Tracker. Health Equity Tracker logo decorative H", which is redundant.